src/checks/conciseness/conciseness.py

```diff
@@ -0,0 +1,113 @@
+from pydantic import ValidationError, BaseModel
+import json
+import openai
+from openai.error import Timeout, APIError, APIConnectionError, InvalidRequestError, AuthenticationError, PermissionError, RateLimitError
+
+from prompt import compare_answers_prompt
+from utils.secret_manager import get_secret
+from utils.logger import get_logger
+from typing import Dict, Union
+import pathlib
+
+logger = get_logger(__name__)
+openai_errors = ( Timeout, APIError, APIConnectionError, InvalidRequestError, AuthenticationError, PermissionError, RateLimitError )
+
+# TODO: we should move these into a different repo, probably llm-toolkit-api, and import them here
```

Comment:
Yes, I agree, this could get pretty complex. Do you know of a good way to ensure that the correct and updated schemas are used, other than sharing a link to the schema in llm_toolkit_api?

Reply:
I don't know any way other than defining schema here.

I don't think it's a big issue, I mean if the API is changing the schema then the logic consuming that schema might also need to be modified so it's fine that they would need to be changed, at the point where they are being used